dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -154,7 +154,7 @@ def test_meta_duplicated():
 
 
 def test_meta_nonempty_empty_categories():
-    for dtype in ['O', 'f8', 'M8']:
+    for dtype in ['O', 'f8', 'M8[ns]']:
```

Comment:
I think this worked by accident previously: https://github.com/pandas-dev/pandas/issues/24753

Not sure if pandas will put in a compat layer for now, or whether we'll just raise.